content/windows/docs/networking.md

```diff
@@ -18,7 +18,7 @@ Using the formula above, we can calculate max pods for an m5.large instance.
 
 _(1 primary ENI * 10 secondary IPs per ENI) - 3 = 7_
 
-The reason it only support 7 is because the CNI only allows us to use the worker node's primary ENI. Also each ENI on an m5.large supports 10 secondary IP addresses, so we subtract 3 from 10 to get our total of 7 IP addresses.
+**Note: The reason it only support 7 is because the VPC CNI for Windows only allows us to use the worker node's primary ENI. Also each ENI on an m5.large supports 10 secondary IP addresses, so we subtract 3 from 10 to get our total of 7 IP addresses.**
```

Comment:
I don't think you need to explain this. It's pretty clear from the formula and the explanation about. The only thing you're really explaining here is that on Windows, we can only use the primary ENI.

Reply:
I changed to make it clear that is the VPC CNI for Windows that imposes this limit, reading the original text it doesn't make much clear on what CNI we are talking about.